src/yield-claimers/YieldClaimer.sol

```diff
@@ -28,37 +28,33 @@ abstract contract YieldClaimer is IActionBase, AbstractBase {
     // The contract address of the Arcadia Factory.
     IArcadiaFactory public immutable ARCADIA_FACTORY;
 
-    // The maximum fee an initiator can set, with 18 decimals precision.
-    uint256 public immutable MAX_FEE;
-
     /* //////////////////////////////////////////////////////////////
                                 STORAGE
     ////////////////////////////////////////////////////////////// */
 
     // The Account to rebalance the fees for, used as transient storage.
     address internal account;
 
-    // A mapping from initiator to rebalancing fee.
-    mapping(address initiator => InitiatorInfo) public initiatorInfo;
+    // A mapping from account to account specific information.
+    mapping(address initiator => AccountInfo) public accountInfo;
```

Comment:
This is from initiator to account info? Not from acc to acc info

Reply:
no from account